code/game/gamemodes/clock_cult/clock_cult.dm

```diff
@@ -76,6 +76,13 @@ This file's folder contains:
 				M.visible_message("<span class='warning'>[M] whirs as it resists an outside influence!</span>")
 			M << "<span class='warning'><b>Corrupt data purged. Resetting cortex chip to factory defaults... complete.</b></span>" //silicons have a custom fail message
 			return 0
+	else if(istype(M, /mob/living/simple_animal/drone))
```

Comment:
I'm aware. I still don't want to fully abandon drone conversion, so I've left some snippets of code in for either admin shenanigans or future use. I've made sure that they'll never trigger naturally. If you _really_ don't want it, I can cut them out.